apps/fp-frontend-app/src/fagsakprofile/FagsakProfileIndex.tsx

```diff
@@ -60,6 +61,7 @@ export const FagsakProfileIndex = ({
   hentOgSettBehandling,
   visSideMeny,
   toggleSideMeny,
+  visUtvidetHistorikk,
```

Comment:
Denne logikken er felles for fleire ting (historikk, notat osv), så bør ha eit anna navn

Reply:
Jeg har nå endret til visUtvidetInnhold